lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -1477,10 +1488,21 @@ void SwiftASTContext::AddExtraClangArgs(std::vector<std::string> ExtraArgs) {
     if (clang_argument.startswith("-Werror")) {
       continue;
     }
+
+    if (clang_argument.empty()) {
+      continue;
+    }
```

Comment:
nit: drop {} here

The LLVM style prefers no curly braces on single statements.